src/OpenSage.Game/Utilities/Sdl2Interop.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
```

Comment:
I think in the long-run we should try to get some of this into Veldrid.SDL, since people request similar improvements there